### PR TITLE
Docs a

### PR DESCRIPTION
**Pull request by bobmcwhirter**
- [ ] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [ ] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [ ] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Review by kenfinnigan — APPROVED** [review]

File: docs/concepts/packaging-types.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+When using WildFly Swarm, you have two different ways to package your
+runtime and application, depending on how you intend to use and deploy
+it.
+
+## Uberjar
+
+An _uberjar_ is a single Java `.jar` file that includes everything that
+is required to execute your application.  This includes both the selected
+runtime components (the parts you may consider to be the _app server_)
+along with the application compoments (you `.war`).
```
Comment (Ladicek) [inline docs/concepts/packaging-types.adoc:13]: "you .war" -> "your .war"?

**Comment by tradej** [comment]
@kenfinnigan, I understand there may be pressure on getting PRs in, but I didn't have time to review the pull request as it was filed after my EOB on Friday, and it contains errors that would be caught in the review. Can you please hold off on merging docs-related PRs for at least 24 hours after they are filed so that they can get my ACK? Thank you.

Filed [SWARM-1598](https://issues.jboss.org/browse/SWARM-1598) to track the review effort.

**Comment by bobmcwhirter** [comment]
Sorrry that was me wanting to land this and then iterate. I like to merge
early and often.

**Comment by tradej** [comment]
All right, @bobmcwhirter, I get where you're coming from, but we should set a sort of an agreement about what is and is not acceptable for the pull request to proceed, so that we can meaningfully synchronise upstream and downstream work. Let's talk more in the weekly meeting.

**Pull request merged**